libs/clients/cms/schema.graphql

```diff
@@ -1,7 +1,10 @@
+directive @contentSourceMaps on QUERY
+
 """
 A slice with accordions [See type definition](https://app.contentful.com/spaces/8k0h54kbe6bj/content_types/accordionSlice)
 """
-type AccordionSlice implements Entry {
+type AccordionSlice implements Entry & _Node {
+  _id: ID!
```

Comment:
Do you need to add `_id` since you've already created an intersection with `_Node` which already has `_id: ID!`. I see it is repeated in several places below